qemu/tests/cfg/qemu_guest_agent_hotplug.cfg

```diff
@@ -15,12 +15,12 @@
     gagent_check_type = sync
     # /qga_setup_fail.log is saved during guest preinstall phase, if you don't have this file, please skip this parameter.
     cmd_check_qga_installlog = "ls /qga_setup_fail.log"
-    extra_params = "-device virtio-serial-pci,id=qga-virtio-serial0,bus=pci.0"
+    extra_params = "-device '{"driver":"virtio-serial-pci","id": "qga-virtio-serial0","bus":"pci.0"'"
```

Comment:
Hi @6-dehan 

We can not convert the command line to the JSON format directly since this action depends on the requirement of the downstream, like the host version or other conditions. We can not give the same rule for the upstream testing.

And there is a root issue is that this case uses the `extra_params` to define the device which is no better way to do that since it will generate the `qdevices.StrDev` instance which is not good for determining a qemu device, we should avoid using the such interface as far as possible.

BTW, it seems like the framework has already supported creating the device `virtio-serial-pci`(https://github.com/avocado-framework/avocado-vt/blob/8e3f75389778f6f9fc7992c166f88814364e0d0f/virttest/qemu_devices/qcontainer.py#L2677), so we can use it and configure the corresponding information in the internal file.

CC @nickzhq  @zhencliu